tests/highlevel/test_messages.py

```diff
@@ -2795,7 +2794,7 @@ def test_basic(self):
 
 class GetExceptionBreakpointEventTests(PyDevdEventTest, unittest.TestCase):
 
-    CMD = CMD_GET_BREAKPOINT_EXCEPTION
+    CMD = 1234
```

Comment:
Is the magic number some leftover?

Reply:
The test is wrongly named. It is supposed to test a unsupported command. I need to change this.

---
In reply to: [223824331](https://github.com/Microsoft/ptvsd/pull/893#discussion_r223824331) [](ancestors = 223824331)

Reply:
I need to update this. There should be two tests here.  One for the GetBreakpointException and another for unsupported command.